client/js/components/ParkMap.jsx

```diff
@@ -1,18 +1,31 @@
+import $ from 'jquery';
+import React from 'react';
+import { GeoJson, Map, Marker, Popup, TileLayer } from 'react-leaflet';
 
-class Park extends React.Component {
+export default class ParkMap extends React.Component {
     constructor(props) {
         super(props);
-        var pID = this.props.parkId;
     }
 
     render () {
+        var center = this.props.center;
+        var p = this.props
```

Comment:
Its easier to read the code if you don't do this. Otherwise I have to look up where you define `p` before I can understand what's going on.